src/style.css

```diff
@@ -1,3 +1,65 @@
+:root {
+  --color1: #00acdc;
+  --color2: #000;
+  --blockheight: 4px;
+  --blocktopgap: 26.5px;
+  --blockwidth: 37px;
+  --border1: 8px;
```

Comment:
Remember that you should create CSS variables only for values that appear more than once!

Also, color1 and color2 are not the most descriptive names